Rematch/UI.lua

```diff
@@ -86,7 +86,7 @@ function RematchPlugin:SetupUI()
     scriptButtonIcon = scriptButtonIcon:gsub('{{REMATCHVERSION}}', rematchVersion.major)
     if rematchVersion >= ns.Version:New(5, 0, 0, 0) then
         -- TODO: restore tooltip/button once Rematch supports that again
-        Rematch.badges:RegisterBadge('teams', 'PetBattleScripts', scriptButtonIcon, nil, function(teamID)
+        Rematch.badges:RegisterBadge('teams', 'PetBattleScripts', scriptButtonIcon, nil, function(_, teamID)
```

Comment:
Looks fine, but maybe we should add compatibility for both by having a version check?

Reply:
I can get that tested and committed later today.